asyncauth.go

```diff
@@ -147,9 +151,9 @@ func renderPluginInstructions(w http.ResponseWriter, data TemplateData) error {
 	return nil
 }
 
-func getDownloadURL(url, platform, version string) string {
+func getDownloadURL(url, platform, version, binary string) string {
 	// TODO: make this more readable
-	return fmt.Sprintf("%s%s%s/%s_%s/%s", url, downloadPath, platform, binaryName, version, binaryName)
+	return fmt.Sprintf("%s%s%s/%s_%s/%s", url, downloadPath, platform, "konvoy-async-auth", version, binary)
```

Comment:
Perhaps we should put what is now the "binary base name" into a constant.

Reply:
:+1: